index.html

```diff
@@ -2740,23 +2740,20 @@ <h3>Zero-Knowledge Proofs</h3>
     "https://www.w3.org/2018/credentials/examples/v1"
   ],
   "type": "VerifiablePresentation",
-  "verifiableCredential": [{
-    "@context": [
-      "https://www.w3.org/2018/credentials/v1",
-      "https://www.w3.org/2018/credentials/examples/v1"
-    ],
-    "type": ["VerifiableCredential", "UniversityDegreeCredential"],
-    <span class="highlight">"credentialSchema": {
-      "id": "did:example:cdf:35LB7w9ueWbagPL94T9bMLtyXDj9pX5o",
-      "type": "did:example:schema:22KpkXgecryx9k7N6XN1QoN3gXwBkSU8SfyyYQG"
-    }</span>,
+  "verifiableCredential": [
+    {
+        "@context": [ "https://www.w3.org/2018/credentials/v1", "https://www.w3.org/2018/credentials/examples/v1" ],
```

Comment:
Use multi-line formatting.  Fits screen better and similar to all other examples.
```suggestion
        "@context": [
          "https://www.w3.org/2018/credentials/v1",
          "https://www.w3.org/2018/credentials/examples/v1"
        ],
```

Reply:
Hey @jonnycrunch, PR looks good after you accept @davidlehn's suggestion. Can you please do that so I can merge this PR?